File: Common/Util/src/common/util/RandomInt.java
package common.util;

import java.util.concurrent.ThreadLocalRandom;

public class RandomInt {

    /**
     * Return a random int between min and max (both bounds inclusive)
     * @param min lower bound
     * @param max higher bound
     */
    public static int randInt(int min, int max) {
        return ThreadLocalRandom.current().nextInt(min, max + 1);
    }
}

File: Games/Dennis/src/de/hsh/dennis/model/Spawn/RandomInt.kt
package de.hsh.dennis.model.Spawn

import java.util.concurrent.ThreadLocalRandom

object RandomInt {
    /**
     * Return a random int between min and max (both bounds inclusive)
     * @param min lower bound
     * @param max higher bound
     */
    fun randInt(min: Int, max: Int): Int {
        return ThreadLocalRandom.current().nextInt(min, max + 1)
    }
}
